src/SSHDebugTests/SSHDebugUnitTests.csproj

```diff
@@ -65,6 +65,11 @@
       <Project>{15bcbef4-1c2b-412b-925b-34a049097e62}</Project>
       <Name>SSHDebugPS</Name>
     </ProjectReference>
+    <ProjectReference Include="..\DebugEngineHost.Stub\DebugEngineHost.Stub.csproj">
```

Comment:
I don't know if this is a bug but it is already declared false in:
https://github.com/microsoft/MIEngine/pull/1075/files#diff-34e5ea0482926d6a8c626f9912d5786f1c996e27b3016be4b1bdcee5c6ff8518R117-R132

This was needed to stop https://dev.azure.com/ms/MIEngine/_build/results?buildId=120543&view=logs&j=9d538be6-44f8-50c3-7930-4e7f88a0f6b2&t=86540a71-1fda-5070-7b33-ef0a45c48f8e&l=971 and make the unit tests work.